### ISSUE TITLE
emsdk install failed with building by VS 2019 on windows

### ISSUE THREAD

**Issue by vincent-unity** [emscripten-core/emsdk#651]
emsdk install failed while building llvm by VS 2019 on windows.
```Exception: Unknown generator used to build solution "..\LLVM.sln```

The problem is in emsdk.py, there is 
```
def get_generator_for_sln_file(sln_file):
  contents = open(sln_file, 'r').read()
  if '# Visual Studio 16' in contents:  # VS2019
    return 'Visual Studio 16'
  if '# Visual Studio 15' in contents:  # VS2017
    return 'Visual Studio 15'
  raise Exception('Unknown generator used to build solution file ' + sln_file) 
```
But the LLVM.sln generated for VS 2019 contains `# Visual Studio Version 16'`. This caused the Unknown generator used to build solution LLVM.sln.

**Comment by sbc100** [comment]
So is this fix to update the string we use to match `VS2019`.

Are you intending to build the SDK from source?  Most emsdk users are able to use our pre-built binaries which saved a lot of time.  Which sdk version are you trying to install?

**Comment by vincent-unity** [comment]
Yes and Yes. I was building the SDK from the source.  Currently we're building 1.39.17.2, but this issue is on https://github.com/emscripten-core/emsdk master too.

**Comment by sbc100** [comment]
Can you upload a patch to fix the string name?

Why are you needing to build from source?
